$OpenBSD: patch-src_librustc_driver_lib_rs,v 1.1 2018/04/02 07:21:50 landry Exp $
Declare "sse2" feature on amd64 to be always present.
See https://github.com/rust-lang/rust/issues/46181
Index: src/librustc_driver/lib.rs
--- src/librustc_driver/lib.rs.orig
+++ src/librustc_driver/lib.rs
@@ -130,6 +130,9 @@ pub mod target_features {
             cfg.insert((tf, Some(feat)));
         }
 
+        #[cfg(all(target_os="openbsd", target_arch="x86_64"))]
+        cfg.insert((tf, Some(Symbol::intern("sse2"))));
+
         if sess.crt_static_feature() {
             cfg.insert((tf, Some(Symbol::intern("crt-static"))));
         }
